### PR TITLE
Added test case for TopK

### PR DESCRIPTION
**Pull request by martinpinto**
Added test case for TopK. The structure of the test case follows the structure of previous test cases.

### CODE REVIEW

File: src/sketches/topk_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	info.Name = utils.Stringp("marvel")
+	sketch, err := NewTopKSketch(info)
+
+	if err != nil {
+		t.Error("expected avengers to have no error, got", err)
+	}
+
+	values := [][]byte{
+		[]byte("sabertooth"),
+		[]byte("thunderbolt"),
```
Comment (seiflotfy) [inline src/sketches/topk_test.go]: Add an extra thunderbolt

File: src/sketches/topk_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	pb "datamodel/protobuf"
+	"utils"
+)
+
+func TestAddTopK(t *testing.T) {
+	utils.SetupTests()
+	defer utils.TearDownTests()
+
+	info := datamodel.NewEmptyInfo()
+	info.Properties.Size = utils.Int64p(1024)
```
Comment (seiflotfy) [inline src/sketches/topk_test.go]: limit to 3

File: src/sketches/topk_test.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+	if res, err := sketch.Get(values); err != nil {
+		t.Error("expected no errors, got", err)
+	} else {
+		tmp := res.(*pb.RankingsResult)
+		rres := tmp.GetRankings()
+		for i := 0; i < len(rres); i++ {
+			for key := range expectedRankings {
+				if rres[i].Value == utils.Stringp(key) && rres[i].Count != utils.Int64p(expectedRankings[key]) {
+					t.Error("expected cardinality == "+strconv.FormatInt(expectedRankings[key], 10)+", got", rres[i].Count)
```
Comment (seiflotfy) [inline src/sketches/topk_test.go]: this is not cardinality

**Comment by seiflotfy** [comment]
The unit test should also check the order of the ranks coming in, where the biggest frquency is at the top

File: src/sketches/topk_test.go (hunk 1 above)
Comment (martinpinto) [inline src/sketches/topk_test.go]: done

**Pull request merged**